### ISSUE TITLE
False-positive E1102:spatial.cKDTree is not callable

### ISSUE THREAD

**Issue by bersbersbers** [PyCQA/pylint#2192]
### Steps to reproduce
```
from scipy import spatial
Tree = spatial.cKDTree([[1, 2], [3, 4]])
print(Tree.query([2, 3]))
```
### Current behavior
E1102

### Expected behavior
No error

### pylint --version output
No config file found, using default configuration
pylint 1.9.2,
astroid 1.6.5
Python 3.6.5 (v3.6.5:f59c0932b4, Mar 28 2018, 17:00:18) [MSC v.1900 64 bit (AMD64)]

**Comment by PCManticore** [comment]
Try passing `--extension-pkg-whitelist=scipy` to pylint, this should fix your problem.
